src/core.js

```diff
@@ -927,7 +927,7 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
     if (isFunction(beforeChangeResult)) {
       warn('Your beforeChange callback returns a function. It\'s not supported since Handsontable 0.12.1 (and the returned function will not be executed).');
 
-    } else if (beforeChangeResult === false) {
+    } else if (beforeChangeResult === false || beforeChangeResult.length === 0 || beforeChangeResult[0] === null) {
```

Comment:
Please check also whether `null` occurs in the rest of `beforeChangeResult` array.

Reply:
In this case we interested only the first input. Because the rest is check further in different place:

```if (changes[i] === null) {
        changes.splice(i, 1);
      } else {```.

But thanks, after improvement we handle all cases like `changes[i] = null;`.